{
  "id": "uno-ooo-parser",
  "version": "0.1.16",
  "libre_office_ver": "7.2",
  "name": "URLAuthenticationRequest",
  "type": "exception",
  "namespace": "com.sun.star.ucb",
  "parser_args": {
    "sort": true,
    "long_names": true,
    "remove_parent_inherited": true
  },
  "writer_args": {
    "include_desc": true
  },
  "data": {
    "from_imports": [
      [
        ".authentication_request",
        "AuthenticationRequest",
        "authentication_request_i"
      ]
    ],
    "from_imports_typing": [],
    "extends_map": {
      "com.sun.star.ucb.AuthenticationRequest": "authentication_request_i"
    },
    "quote": [],
    "typings": [],
    "requires_typing": false,
    "name": "URLAuthenticationRequest",
    "namespace": "com.sun.star.ucb",
    "imports": [],
    "extends": [
      "com.sun.star.ucb.AuthenticationRequest"
    ],
    "desc": [
      "An error specifying lack of correct authentication data (e.g., to log into an account).",
      "",
      "**since**",
      "",
      "    OOo 3.2"
    ],
    "url": "https://api.libreoffice.org/docs/idl/ref/exceptioncom_1_1sun_1_1star_1_1ucb_1_1URLAuthenticationRequest.html",
    "items": {
      "properties": [
        {
          "name": "URL",
          "returns": "str",
          "desc": [
            "The URL for which authentication is requested."
          ],
          "raises_get": "",
          "raises_set": ""
        }
      ]
    }
  }
}